### PR TITLE
RELATED: ONE-3336 ONE-3361 Better axis line handling

### PR DESCRIPTION
**Pull request by peterlacko**

### CODE REVIEW

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/1610/ : SUCCESS in 7s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/1084/ : FAILURE in 1m 36s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/1611/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/1085/ : FAILURE in 3m 50s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/1620/ : SUCCESS in 8s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/1090/ : FAILURE in 58s

**Comment by peterlacko** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/1625/ : SUCCESS in 8s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/1095/ : SUCCESS in 3m 53s

File: src/components/visualizations/chart/highcharts/customConfiguration.ts [hunk 1]
```diff
@@ -828,13 +829,23 @@ function shouldExpandYAxis(chartOptions: any) {
     return min === '' && max === '' ? {} : { getExtremesFromAll: true };
 }
 
-function isAxisLineVisible(isAxisVisible: boolean, isGridEnabled: boolean) {
-    return isAxisVisible && !isGridEnabled;
+function getAxisLineConfiguration(chartType: ChartType, isAxisVisible: boolean) {
+    const lineColor = '#d5d5d5';
```
Comment (ZdenekVidensky) [inline src/components/visualizations/chart/highcharts/customConfiguration.ts]: I would move this line color into colors.js utils file as a constant

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/1630/ : SUCCESS in 10s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/1098/ : SUCCESS in 2m 51s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/355/ : SUCCESS in 1m 31s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/1099/ : SUCCESS in 13m 06s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/640/ : SUCCESS in 27m 14s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/270/ : SUCCESS in 15m 15s

**Pull request merged**